k2/csrc/ragged_ops.h

```diff
@@ -427,6 +412,14 @@ Ragged<int32_t> AddPrefixToRagged(Ragged<int32_t> &src,
  */
```

Comment:
> Read algorithms like in compose.cc to understand why.

What do you mean by saying `compose.cc`? There is no such file in k2.

Reply:
can remove this sentence, must have moved things around.